### PR TITLE
[improvement](group_comit) Add bvar to monitor the total wal count on disk

### PR DESCRIPTION
**Pull request by hust-hhb**
## Proposed changes
Use a bvar to monitor the total wal count on disk.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hust-hhb** [comment]
run buildall

File: be/src/olap/wal/wal_manager.cpp [hunk 1]
```diff
@@ -380,6 +386,8 @@ Status WalManager::_replay_background() {
         if (_first_replay.compare_exchange_strong(expected, false)) {
             RETURN_IF_ERROR(_load_wals());
         }
+        auto _total_wal_size = get_wal_queue_size(-1);
+        g_wal_total_count.set_value(_total_wal_size);
```
Comment (mymeiyi) [inline be/src/olap/wal/wal_manager.cpp:390]: can write to one line:
g_wal_total_count.set_value(get_wal_queue_size(-1));

Comment (hust-hhb) [reply be/src/olap/wal/wal_manager.cpp]: @mymeiyi already change

**Review by mymeiyi — APPROVED** [review]

**Comment by hust-hhb** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run p0

**Comment by hust-hhb** [comment]
run p0

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Pull request merged**